fuzz/Cargo.toml

```diff
@@ -19,9 +19,9 @@ cargo-fuzz = true
 
 [features]
 default = ["native"]
-native = ["libfuzzer-sys", "zstd", "vortex-file"]
+native = ["libfuzzer-sys", "zstd", "vortex-file", "vortex/files"]
 wasmfuzz = []
-zstd = ["vortex-layout/zstd"]
+zstd = ["vortex/zstd"]
```

Comment:
In addition to `vortex/files` feature, this is required to enable zstd encoding.